frontend/src/components/maps/leaflet/Map.tsx

```diff
@@ -329,6 +320,14 @@ const Map: React.FC<MapProps> = ({
 
   const [infoOpen, setInfoOpen] = React.useState(false);
   const [layersOpen, setLayersOpen] = React.useState(false);
+  const displayMessage = (
```

Comment:
Nice simple fix.  I like it.  Handles both scenarios.  We can look into a better one later.